### PR TITLE
Fix grammatical error

### LINKED ISSUE #5127: Grammatical error
The Note shows a grammatical error:

You'll need to confirm the Linux distributions and versions of Android, iOS and macOS **you've are** compatible with Defender for Endpoint for the integration to work.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b4b36b3a-e292-e035-ab94-895c56b46c76
* Version Independent ID: 2e02900f-f545-a199-8d1c-b6a7958ee1df
* Content: [Minimum requirements for Microsoft Defender for Endpoint](https://docs.microsoft.com/en-us/microsoft-365/security/defender-endpoint/minimum-requirements?view=o365-worldwide)
* Content Source: [microsoft-365/security/defender-endpoint/minimum-requirements.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender-endpoint/minimum-requirements.md)
* Product: **m365-security**
* Technology: **mde**
* GitHub Login: @mjcaparas
* Microsoft Alias: **macapara**

### PR DESCRIPTION
**Pull request by konstruktoid**
- fix grammatical error
- basic linting

closes #5127

Signed-off-by: Thomas Sjögren <konstruktoid@users.noreply.github.com>

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/defender-endpoint/minimum-requirements.md [hunk 1]
```diff
@@ -83,6 +86,7 @@ Access to Defender for Endpoint is done through a browser, supporting the follow
 ## Hardware and software requirements
 
 ### Supported Windows versions
+
 - Windows 7 SP1 Enterprise ([Requires ESU for support](https://docs.microsoft.com/troubleshoot/windows-client/windows-7-eos-faq/windows-7-extended-security-updates-faq).)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/minimum-requirements.md]:
```suggestion
- Windows 7 SP1 Enterprise ([Requires ESU for support](/troubleshoot/windows-client/windows-7-eos-faq/windows-7-extended-security-updates-faq).)
```

File: microsoft-365/security/defender-endpoint/minimum-requirements.md [hunk 2]
```diff
@@ -83,6 +86,7 @@ Access to Defender for Endpoint is done through a browser, supporting the follow
...
 - Windows 7 SP1 Pro ([Requires ESU for support](https://docs.microsoft.com/troubleshoot/windows-client/windows-7-eos-faq/windows-7-extended-security-updates-faq).)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/minimum-requirements.md]:
```suggestion
- Windows 7 SP1 Pro ([Requires ESU for support](/troubleshoot/windows-client/windows-7-eos-faq/windows-7-extended-security-updates-faq).)
```

File: microsoft-365/security/defender-endpoint/minimum-requirements.md [hunk 3]
```diff
@@ -113,17 +117,19 @@ The hardware requirements for Defender for Endpoint on devices are the same for
...
 ### Other supported operating systems
+
 - [Android](microsoft-defender-endpoint-android.md)
 - [iOS](microsoft-defender-endpoint-ios.md)
 - [Linux](microsoft-defender-endpoint-linux.md)
 - [macOS](microsoft-defender-endpoint-mac.md)
 
 > [!NOTE]
-> You'll need to confirm the Linux distributions and versions of Android, iOS and macOS you've are compatible with Defender for Endpoint for the integration to work.
+> You'll need to confirm the Linux distributions and versions of Android, iOS and macOS are compatible with Defender for Endpoint for the integration to work.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/minimum-requirements.md]:
```suggestion
> You'll need to confirm the Linux distributions and versions of Android, iOS, and macOS are compatible with Defender for Endpoint for the integration to work.
```

File: microsoft-365/security/defender-endpoint/minimum-requirements.md [hunk 4]
```diff
@@ -193,6 +200,7 @@ Before you onboard devices, the diagnostic data service must be enabled. The ser
 
 
 ## Microsoft Defender Antivirus configuration requirement
+
 The Defender for Endpoint agent depends on the ability of Microsoft Defender Antivirus to scan files and provide information about them.
 
 Configure Security intelligence updates on the Defender for Endpoint devices whether Microsoft Defender Antivirus is the active antimalware or not. For more information, see [Manage Microsoft Defender Antivirus updates and apply baselines](https://docs.microsoft.com/windows/security/threat-protection/microsoft-defender-antivirus/manage-updates-baselines-microsoft-defender-antivirus).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/minimum-requirements.md]:
```suggestion
Configure Security intelligence updates on the Defender for Endpoint devices whether Microsoft Defender Antivirus is the active antimalware or not. For more information, see [Manage Microsoft Defender Antivirus updates and apply baselines](/windows/security/threat-protection/microsoft-defender-antivirus/manage-updates-baselines-microsoft-defender-antivirus).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @konstruktoid, although these suggestions are outside the scope of this PR, could you please commit my suggested changes (mainly changing links to relative links). Could you also change the link in line 212 to /windows/security/threat-protection/microsoft-defender-antivirus-compatibility Many thanks.

**Comment by konstruktoid** [comment]
Thanks @JohanFreelancer9!

File: microsoft-365/security/defender-endpoint/minimum-requirements.md [hunk 5]
```diff
@@ -193,26 +200,29 @@ Before you onboard devices, the diagnostic data service must be enabled. The ser
...
 
 If your organization has turned off Microsoft Defender Antivirus through group policy or other methods, devices that are onboarded must be excluded from this group policy.
 
-If you're onboarding servers and Microsoft Defender Antivirus isn't the active antimalware on your servers, Microsoft Defender Antivirus will either need to be configured to go on passive mode or uninstalled. The configuration is dependent on the server version. For more information, see [Microsoft Defender Antivirus compatibility](https://docs.microsoft.com/windows/security/threat-protection/microsoft-defender-antivirus-compatibility.md).
+If you're onboarding servers and Microsoft Defender Antivirus isn't the active antimalware on your servers, Microsoft Defender Antivirus will either need to be configured to go on passive mode or uninstalled. The configuration is dependent on the server version. For more information, see [Microsoft Defender Antivirus compatibility](/security/defender-endpoint/microsoft-defender-antivirus-compatibility.md).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/minimum-requirements.md]:
```suggestion
If you're onboarding servers and Microsoft Defender Antivirus isn't the active antimalware on your servers, Microsoft Defender Antivirus will either need to be configured to go on passive mode or uninstalled. The configuration is dependent on the server version. For more information, see [Microsoft Defender Antivirus compatibility](/security/defender-endpoint/microsoft-defender-antivirus-compatibility).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@konstruktoid - Many thanks, please just remove the extension in the link in line 212 as per my suggestion. Thanks again.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Pull request merged**